app/controllers/standup_meeting_groups/joins_controller.rb

```diff
@@ -37,6 +36,9 @@ def destroy
           helpers.dom_id(@standup_meeting_group, :join_or_leave), component
         )
       end
+      format.html do
```

Comment:
since we're only enabling the leaving of a standup meeting group from the show page, it should be an html request